src/doc/src/reference/config.md

```diff
@@ -1058,3 +1061,16 @@ Sets the width for progress bar.
 [revision]: https://git-scm.com/docs/gitrevisions
 [registries]: registries.md
 [crates.io]: https://crates.io/
+
+### `[future-incompat-report]`
```

Comment:
Can this be placed alphabetically? That is, place this just above the `[env]` section.